scripts/globals/abilities/enmity_douse.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Ability: Enmity Douse
+-- Description Reduces the target's enmity towards you.
+-- Obtained: BLM Level 87
+-- Recast Time: 0:10:00
+-----------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+-----------------------------------
+
+function onAbilityCheck(player,target,ability)
+    return 0,0
+end
+
+function onUseAbility(player,target,ability)
+    if (target:isMob()) then
```

Comment:
I don't think an automated tool will handle this sort of thing, but simple logic statements can just be `if condition then` without any additional parens

Reply:
took care of this among others